src/Controllers/SchemaController.js

```diff
@@ -76,7 +76,7 @@ const defaultColumns = Object.freeze({
     "pushTime":     {type:'String'},
     "source":       {type:'String'}, // rest or webui
     "query":        {type:'String'}, // the stringified JSON query
-    "payload":      {type:'Object'}, // the JSON payload,
+    "payload":      {type:'String'}, // the stringified JSON payload,
```

Comment:
Not really, it's set as a string already in the pushStatusHandler: 
See: 
https://github.com/ParsePlatform/parse-server/blob/master/src/pushStatusHandler.js#L27
(https://github.com/ParsePlatform/parse-server/blob/master/src/pushStatusHandler.js#L41)